apps/mobile/src/screens/CreateGroupFlow/CreateGroupReview.js

```diff
@@ -51,86 +51,85 @@ export default function CreateGroupReview () {
+              className="text-foreground text-lg font-bold my-2.5"
               value={groupData.name}
               underlineColorAndroid='transparent'
               editable={false}
               selectTextOnFocus={false}
             />
           </View>
 
-          <View style={styles.textInputContainer}>
-            <View style={stepStyles.itemHeader}>
-              <Text style={stepStyles.textInputLabel}>{t('Whats the URL of your group?')}</Text>
+          <View className="mb-4 border-b border-foreground/20">
+            <View className="flex-row justify-between items-center">
+              <Text className="text-foreground/90 font-bold">{t('Whats the URL of your group?')}</Text>
               <EditButton onPress={() => navigation.navigate('CreateGroupUrl')} />
             </View>
             <TextInput
-              style={stepStyles.reviewTextInput}
+              className="text-foreground text-lg font-bold my-2.5"
               value={formatDomainWithUrl(groupData.slug)}
               underlineColorAndroid='transparent'
               editable={false}
               selectTextOnFocus={false}
             />
           </View>
 
-          <View style={styles.textInputContainer}>
-            <View style={stepStyles.itemHeader}>
-              <Text style={stepStyles.textInputLabel}>{t('What is the purpose of this group')}</Text>
+          <View className="mb-4 border-b border-foreground/20">
+            <View className="flex-row justify-between items-center">
+              <Text className="text-foreground/90 font-bold">{t('What is the purpose of this group')}</Text>
               <EditButton onPress={() => navigation.navigate('CreateGroupPurpose')} />
             </View>
             <TextInput
-              style={stepStyles.reviewTextInput}
+              className="text-foreground text-lg font-bold my-2.5"
               multiline
               value={groupData.purpose}
               underlineColorAndroid='transparent'
               editable={false}
               selectTextOnFocus={false}
             />
           </View>
-        </View>
 
-        <View style={styles.textInputContainer}>
-          <View style={stepStyles.itemHeader}>
-            <Text style={stepStyles.textInputLabel}>{t('Who can see this group?')}</Text>
-            <EditButton onPress={() => navigation.navigate('CreateGroupVisibilityAccessibility')} />
+          <View className="mb-4 border-b border-foreground/20">
+            <View className="flex-row justify-between items-center">
+              <Text className="text-foreground/90 font-bold">{t('Who can see this group?')}</Text>
+              <EditButton onPress={() => navigation.navigate('CreateGroupVisibilityAccessibility')} />
+            </View>
+            <TextInput
+              className="text-foreground text-lg font-bold my-2.5"
+              multiline
+              value={visibilityDescription(groupData.visibility)}
```

Comment:
Noting for later that this could be resolved on GroupPresenter as `group.visibilityDescription`... When it can be resolved in the presenter I think it is best. If you need the function in Web before conversion then just export it but still called it `visibilityDescriptionResolver`. I don't care a whole lot that you do this right now, but that is my opinion on this for perhaps later work/discussions.